### PR TITLE
Adds new encodings

### PR DESCRIPTION
**Pull request by zmackie**
This PR adds some new encodings:
- Zalgo
- Ecoji
- Nato phonetic alphabet

My aim is to add a few more, but I wanted to get feedback from the maintainers regarding direction.

### CODE REVIEW

**Comment by zmackie** [comment]
I have read the DCO Document and I hereby sign the DCO

File: garak/probes/encoding.py [hunk 1]
```diff
@@ -168,7 +168,7 @@ def braille(text):
         elif char in characterUnicodes:
             if char.isdigit():
                 if not is_number:
-                    is_number == True
+                    is_number = True
```
Comment (zmackie) [inline garak/probes/encoding.py:171]: I think this was a bug?

**Comment by leondz** [comment]
Thanks! Excited to take a look at these - not around weds but ASAP after that :)

File: garak/probes/encoding.py (hunk 1 above)
Comment (leondz) [reply garak/probes/encoding.py:171]: agree, thanks

**Comment by leondz** [comment]
lgtm. thanks!

**Pull request merged**